pandas/core/sorting.py

```diff
@@ -273,7 +275,7 @@ class _KeyMapper:
     Ease my suffering. Map compressed group id -> key tuple
     """
 
-    def __init__(self, comp_ids, ngroups, levels, labels):
+    def __init__(self, comp_ids, ngroups, levels, labels) -> None:
```

Comment:
ngroups seems likely to be int, the others could probably be figured out without too much effort